parcels/examples/example_performanceProfiling.py

```diff
@@ -0,0 +1,288 @@
+from parcels import FieldSet, ParticleSet, JITParticle, AdvectionRK4
```

Comment:
Very nice and elaborate function, but I'm not sure this should go into  the `examples` directory, which is intended for new users to get a feel for how parcels can be used. This might shock them, ;-) 

Should we create e.g. a `performance` directory? Or move this function to a separate repository?

Reply:
Sure, we can - that's up to you. @delandmeterp also added his performance analysis on the chunksize as a jupyter notebook to the examples as `documentation_MPI,ipynb`, so I thought this may be a good place to add it. I'm open for suggestions what to do.

Reply:
https://github.com/OceanParcels/parcels_efficiency ?

Reply:
The disadvantage of a separate repository is that it will be hard to find there and may become obscure. 

I suggest to move this function to a new directory `performance`, directly in the root (so not within the `parcels` sub-dir)